src/server/api/controllers/profile.controller.js

```diff
@@ -17,6 +17,27 @@ const getStudentsProfile = async (userId) => {
   }
 };
 
+const getAdminsProfile = async (userId) => {
+  /* SQL query to get all the admins 
+  SELECT users.name, roles.name
+  FROM users 
+  join user_roles on users.id = user_roles.user_id 
+  join roles on user_roles.role_id = roles.id
+  where roles.name= "admin";
+  */
+  try {
+    const profile = await knex('users')
```

Comment:
This variable should be named `profiles` since it returns an array.

Reply:
yes true, knex returns array of object, even though result is one object